app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
`||=` giải thích cách toán tử này hoạt động

Reply:
toán tử này sẽ kiểm tra biến nếu như có là `nil` hoặc `false` thì sẽ gán giá trị cho biến bằng giá trị sau dấu `=`
ở trong trường hợp này, sẽ kiểm tra @current_user đã có giá trị hay chưa? nếu `@current_user` có giá trị rồi thì ko làm gì cả. Nếu `@current_user = nil` hoặc `@current_user = false` thì sẽ gán nó bằng user tìm được thông qua User.find_by ạ. Theo như e hiểu là vậy ^^